src/docs/server/index.md

```diff
@@ -51,6 +51,18 @@ Docker is not required, but recommended! AppVeyor has 1st-class integration with
 
 AppVeyor Server can be installed on **Windows 7 SP1 or later** and **Windows Server 2008 R2 SP1 or later**.
 
+#### Chocolatey Installation
+
+[Install Chocolatey package manager](https://chocolatey.org) if not already installed.
+
+Run the following command:
+
+    choco install appveyor-server
+
+Once AppVeyor service is started, open `http://localhost:8050` in your browser to continue AppVeyor setup.
```

Comment:
This line is not needed as browser window will be opened automatically as with manual installation. I've just checked. Maybe we should add the same wording as below.